### ISSUE TITLE
Link text doesn't meet visibility standards when background colour is blue

### ISSUE THREAD

**Issue by MaddyGuthridge** [MaddyGuthridge/Minifolio#111]
![Image](https://github.com/user-attachments/assets/71a41e54-bec1-4a83-a899-fb078d2929f4)
